mlrun/projects/pipelines.py

```diff
@@ -1081,6 +1081,7 @@ def load_and_run(
         raise RuntimeError(f"Workflow {workflow_log_message} failed") from run.exc
 
     if wait_for_completion:
+        run.wait_for_completion()
```

Comment:
worth wrapping in try/except to ensure that following lines would be executed in case of an error